common/common-client/src/main/java/org/ow2/proactive/authentication/LDAPLoginModule.java

```diff
@@ -29,8 +29,7 @@
 import java.security.SecureRandom;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
-import java.util.Hashtable;
-import java.util.Map;
+import java.util.*;
```

Comment:
import whole util package ?

Reply:
probably because I added some classes for tests that I removed after, but it's not a big deal